File: src/main/java/com/jradams/melophobia/entity/backing/ArtistType.java
package com.jradams.melophobia.entity.backing;

import lombok.Getter;

@Getter
public enum ArtistType {

    BAND("Band"),
    CHOIR("Choir"),
    ORCHESTRA("Orchestra"),
    OTHER("Other"),
    PERSON("Person"),
    SUPERGROUP("Supergroup");

    private final String type;

    ArtistType(String type) {
        this.type = type;
    }
}

File: Service/Data/Enum/ArtistType.cs
namespace Melophobia.Data.Enum
{
    using NpgsqlTypes;

    public enum ArtistType
    {
        [PgName("Choir")]
        Choir,

        [PgName("Group")]
        Group,

        [PgName("Orchestra")]
        Orchestra,

        [PgName("Other")]
        Other,

        [PgName("Person")]
        Person
    }
}
